contracts/interfaces/ITellorIIV.sol

```diff
@@ -1,6 +1,23 @@
 pragma solidity ^0.5.0;
 
 contract ITellorIIV{
+
+    event NewTellorAddress(address _newTellor);
```

Comment:
Was this auto-generated or manual?

Reply:
No, I actually created this one myself. There's no good tools that I know of to generate this type of file